site/gatsby-site/cypress.config.js

```diff
@@ -1,6 +1,11 @@
 const { defineConfig } = require('cypress');
 
+require('dotenv').config();
+
+const config = require('./config');
```

Comment:
Cypress already does this. It is inside `cypress/plugins/index.js` 🤔

Reply:
Done!